### ISSUE TITLE
Upcoming: Hue api reworks for bridge v2

### ISSUE THREAD

**Issue by Moyster** [mpolednik/script.kodi.hue.ambilight#80]
https://developers.meethue.com/entertainment-blog  
  
tl;dr : udp rework : The need for a time synchronization-driven interface led us towards a new bridge-streaming API for Hue Entertainment. This bridge API is based on UDP and DTLS rather than HTTP, allowing us to drop frames to maintain synchronization as necessary.  
- sdk : Called the Hue Entertainment Development Kit (EDK), it is written in C++ with portability and usability in the gaming world in mind.  
- spatialisation : the EDK takes care of mapping spatial effects onto the user’s specific entertainment layout, which is created in the Hue app.

beta sub program : https://developers.meethue.com/entertainment
  
This is "limited" to v2 bridges tho, which spawned a few questions / thoughts for later about this plugin :  
- dropping v1 bridge support ? Supporting both ? Supporting a v16 / v17+ branches with each a legacy/v2 branch ? (send 1 / 2 / 3 / 4 at 888-HUEHUE )  
- thoughts about the news ?  

Cheers :)

**Comment by mpolednik** [comment]
Cool find!

And awesome. Except for one thing - I'm still running on the "ancient" v1 bridge. :(

Will try to get the bridge and reserve some time off to start working on that & finally clean up the issue backlog.

**Comment by Moyster** [comment]
Try to lurk ebay or the r/hue subreddit, people often buy a starter pack when they're on sale for 100€~ only for the lightbulbs (instead of 3x45€ at best), they then sell for cheap the extra hub. I guess you get extra credit score if that's for the plugin :>

**Comment by ziggy73701** [comment]
As a side note... if you are willing to keep this going.. I am happy to donate to a fund to get you a new bridge  👯‍♂️

**Comment by joostwestra** [comment]
Indeed I think many people would like to donate if that is the blocking issue. I think it would transform the experience.
